content/doc/developer/testing/index.adoc

```diff
@@ -367,7 +367,19 @@ report.copyFrom(getClass().getResourceAsStream("lint-results_r20.xml"));
 ----
 
 ==== Customizing the `JENKINS_HOME` Directory
-TODO: Write this section.
+
+If you want to adjust the `JENKINS_HOME` within a test, you can make use of the `JenkinsRule`-methods `withExistingHome()` or `withNewHome()`.
```

Comment:
These methods seem to be rarely used.  A GitHub search showed only two usages in the jenkinsci organization.

* [LogStash plugin](https://github.com/jenkinsci/logstash-plugin/blob/a10e30d246c9b51919c4e741d915f23fc7353367/src/test/java/jenkins/plugins/logstash/LogstashBuildWrapperConversionTest.java#L29-L32)
* [LDAP email plugin](https://github.com/jenkinsci/ldapemail-plugin/blob/05051099ee9f56a530ca82d1ef5395c07026737d/src/test/java/com/mtvi/plateng/hudson/ldap/GoodConfigLoadTest.java#L26-L29)

A minor phrasing change.  I'm not sure of the preferred replacement for these test methods, but I suspect that there are preferred replacements..

```suggestion
If you want to adjust the `JENKINS_HOME` within a test, use the `JenkinsRule`-methods link:https://javadoc.jenkins.io/component/jenkins-test-harness/org/jvnet/hudson/test/JenkinsRule.html#withExistingHome(java.io.File)[`withExistingHome()`] or link:https://javadoc.jenkins.io/component/jenkins-test-harness/org/jvnet/hudson/test/JenkinsRule.html#withNewHome()[`withNewHome()`].
```